src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/AceEditorBackgroundLinkHighlighter.java

```diff
@@ -147,7 +153,22 @@ public void execute()
          {
             TextFileType fileType = editor_.getFileType();
             highlighters_.clear();
-            highlighters_.add(webLinkHighlighter());
+            if (pUserPrefs_.get().highlightWebLink().getValue())
+	    {
+               highlighters_.add(webLinkHighlighter());
```

Comment:
I think you can remove this line entirely since `bind()` will fire an event for the initial value of the preference.

Reply:
Fixed in 70cd3b7.